gazebo/common/HeightmapData.hh

```diff
@@ -82,12 +83,33 @@ namespace gazebo
       public: static HeightmapData *LoadTerrainFile(
           const std::string &_filename);
 
+      /// \brief Same as LoadTerrainFile with filename, but also
+      /// considers the surface type in the supplied Spherical Coordinates.
+      /// \param[in] _filename The path to the terrain file.
+      /// \param[in] _sphericalCoordinates SphericalCoordinatesPtr from the
+      /// world.
+      /// \return 0 when the operation succeeds to load a file or -1 when fails.
```

Comment:
this doc-string and the `\return` in the other overload just above are incorrect. These functions return a pointer to memory allocated to store the heightmap data (and passes ownership of the data along with it) when the operation is successful or `nullptr` if it is not successfully loaded

Reply:
Done in https://github.com/gazebosim/gazebo-classic/pull/3258/commits/3b954aa19fbb64250466f861a69289b55ca7c853, fixed the ``return`` statement in a couple of other places as well.